src/libs/ReportUtils.ts

```diff
@@ -3096,7 +3096,7 @@ function canEditMoneyRequest(reportAction: OnyxInputOrEntry<ReportAction<typeof
     }
 
     if (policy?.type === CONST.POLICY.TYPE.CORPORATE && moneyRequestReport && isSubmitted && isCurrentUserSubmitter(moneyRequestReport.reportID)) {
-        const isForwarded = PolicyUtils.getSubmitToAccountID(policy, moneyRequestReport.ownerAccountID ?? -1) !== moneyRequestReport.managerID;
```

Comment:
Looks like there's a lint error above (line 1298) where another call to PolicyUtils.getSubmitToAccountID needs to get updated